app/mobile-app/src/storybook/stories/Button/Button.stories.js

```diff
@@ -0,0 +1,20 @@
+import { action } from '@storybook/addon-actions';
```

Comment:
Appreciate that these are probably examples that come from the setup by default... but since these Storybook entries are all written in `.js`, will this setup work with TypeScript?

Reply:
I haven't personally done this, but this tutorial is essentially what we went through, and does say

"You don't have to add any packages or configuration for writing your stories in TypeScript; just use the .stories.tsx extension."

When @tvcologan is back next weekend we'll make some progress on it and sort out any weirdness.

meanwhile maybe we get rid of the examples. :)